packages/bindings/src/mocks/mock_queriers.rs

```diff
@@ -50,13 +50,19 @@ impl MockDesmosQuerier {
     pub fn new(balances: &[(&str, &[Coin])]) -> Self {
         MockDesmosQuerier {
```

Comment:
Can you extend the `build` job of the `desmos_bindings.yaml` workflow so that we can catch this errors with the CI?
I think that adding a step thats compiles with the `mocks` feature should be enougth.

Reply:
Cool, great suggestion. Added.